lab1/yoda.html

```diff
@@ -1,9 +1,17 @@
 <!DOCTYPE html>
 <html>
     <head>
-        
+        <link rel='shortcut icon' type='image/png' href='img/favicon.ico'>
+        <title>Yoda Page</title>
+        <script type="text/javascript" src="response_handler.js"></script>
     </head>
     <body>
 
+        <center>
+            <img id="image_box" src="img/regular-force.jpg" alt="Yoda Image"></img>
+            <p id="text">To yoda chat I welcome you. Yes, hmmmmm.</p>
+            <input id="input"></input> <br><br>
```

Comment:
It does not seem like part E6 of the lab (respond when pressing enter") is implemented.